azure-devops/azext_devops/dev/repos/policy.py

```diff
@@ -578,7 +578,7 @@ def create_policy_case_enforcement(repository_id, blocking, enabled,
         detect=detect, organization=organization, project=project)
     policy_client = get_policy_client(organization)
     configuration = create_configuration_object(repository_id, None, blocking, enabled,
-                                                '40e92b44-2fe1-4dd6-b3d8-74a9c21d0c6e',
+                                                '7ed39669-655c-494e-b4a0-a08b4da0fcce',
```

Comment:
Can you change the GUID for update call as well?
https://github.com/Azure/azure-devops-cli-extension/blob/d5b161b357f92d9639415d3bef489a09a6239d46/azure-devops/azext_devops/dev/repos/policy.py#L605